docs/nidcpower/session.rst

```diff
@@ -360,6 +360,8 @@ nidcpower.Session
    +------------------------------------------------------------+
    | :py:func:`reset_with_defaults`                             |
    +------------------------------------------------------------+
+   | :py:func:`self_cal`                                        |
```

Comment:
this addition is not noted in CHANGELOG.md

Reply:
Added